### DISCUSSION TITLE
PhotoPrism on Windows will not search additional directories

### DISCUSSION THREAD

**Discussion by L1ttl3J1m** [photoprism/photoprism#3971] [Q&A]
Setting up the storage volumes like this - which the documentation says is the right way to do it;

volumes:
      - "~/Pictures:/photoprism/originals"                 # original media files (photos and videos)
      - "D:/Phones/DCIM:/photoprism/originals/Phones"
      - "F:/Cameras:/photoprism/originals/Cameras"
      - "G:/Experiences:/photoprism/originals/Experiences"

Does not work. All I get is the files in the ~/Pictures directory, not the other drives.

**Comment by lastzero** [comment]
Do you use the latest versions of Docker Desktop and WSL 2? What Windows version are you using?

**Reply by L1ttl3J1m** [reply]
Yes, latest versions of Docker Desktop - 4.26.1 (131620) and WSL 2. Windows 10 Pro 22H2
